### PR TITLE
Added Indonesia Translation (with some additions) in some pages.

### PR DESCRIPTION
**Pull request by GNZTMPZ**
~~I'll brief what's changed, or just track the commit to see what's changed. (still wip though, do not merge)~~

What's changed:

- move some files to designated place (it was at the wrong place, oof)
- translate some "scripting/functions" to indonesian
- some minor typofix
- and another small details that i forgot to add here.. just look at the commit history 😅 

If @Aiuraa accept my PR, feel free to merge this :>

### CODE REVIEW

**Review by leonardssh — APPROVED** [review]

**Comment by Aiuraa** [comment]
@leonardssh how can i review changes if you already aprove it :\

He ask me on discord to review the changes......

**Comment by leonardssh** [comment]
My approval doesn't prevent you from reviewing. @Aiuraa

**Comment by GNZTMPZ** [comment]
I still working on translating, i just need some advice though. pls do not merge this, *yet*!

I'll ping a member if it's done. themk :>

**Comment by Aiuraa** [comment]
> My approval doesn't prevent you from reviewing. @Aiuraa

It does @leonardssh, since i don't updated CODEOWNERS, when someone approved the change the option is not available anymore.

File: docs/translations/id/scripting/functions/BanEx.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    }
+    return 0;
+}
+```
+
+## Notes
+
+:::warning
+
+Pada versi SA-MP 0.3x, tindakan apa pun yang diambil langsung sebelum Ban() (seperti mengirim pesan dengan SendClientMessage) tidak akan sampai ke player. Timer harus digunakan untuk menunda ban.
```
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/BanEx.md:66]: > Timer harus digunakan untuk menunda ban.

Gak perlu pakai titik, bisa pakai koma dan tambahkan seperti:
", oleh karena itu timer harus digunakan untuk menunda pelarangan"

File: docs/translations/id/scripting/functions/Kick.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+title: Kick
+description: Kick player dari server. Player harus keluar dari game dan re-connect jika mereka masih ingin lanjut bermain.
+tags: ["administration"]
+---
+
+## Deskripsi
+
+Kick player dari server. Player harus keluar dari game dan re-connect jika mereka masih ingin lanjut bermain.
```
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/Kick.md:9]: Deskripsi "Kick" bisa diganti dengan "Tendang"

File: docs/translations/id/scripting/functions/Kick.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+| Nama     | Deskripsi                     |
+| -------- | ----------------------------- |
+| playerid | ID player yang ingin di kick. |
```
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/Kick.md:13]: "di kick" bisa diganti menjadi "ditendang"

File: docs/translations/id/scripting/functions/Kick.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+```c
+// Untuk memunculkan pesan (contoh: alasan) untuk player sebelum koneksi terputus
+// Anda harus menggunakan timer untuk membuat sebuah delay. Delay ini hanya membutuhkan beberapa milisekon saja,
+// akan  tetapi contoh ini menggunakan satu detik penuh hanya untuk berjaga-jaga.
+
+public OnPlayerCommandText(playerid, cmdtext[])
+{
+    if (strcmp(cmdtext, "/kickme", true) == 0)
+    {
+        // Kick player yang menggunakan command ini.
```
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/Kick.md:38]: Ini juga diganti jadi "Tendang", tidak perlu menggunakan "kick"

File: docs/translations/id/scripting/functions/Kick.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+        // Pertama, kirim player nya sebuah pesan.
+        SendClientMessage(playerid, 0xFF0000FF, "You have been kicked!");
+
+        // Mem-kick player nya sedetik kemudian menggunakan timer.
```
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/Kick.md:43]: Komentar ini harusnya bisa diganti menjadi:

"Kemudian, buat sebuah timer untuk menunda fungsi Kick selama sedetik"

File: docs/translations/id/scripting/functions/tolower.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+title: tolower
+description: Function ini mengubah sebuah huruf menjadi huruf kecil.
+tags: ["string"]
+---
+
+<LowercaseNote />
+
+## Deskripsi
+
+Function ini mengubah sebuah huruf menjadi huruf kecil.
```
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/tolower.md]: "Function" dapat diubah menjadi "Fungsi"

File: docs/translations/id/scripting/functions/tolower.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+{
+    text[0] = tolower(text[0]);
+    //Ini mengubah karakter utama menjadi huruf kecil.
+    return 1;
+}
+```
+
+## Fungsi Terkait
+
+- [toupper](toupper)
```
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/tolower.md:34]: Deskripsinya jangan lupa.

File: docs/translations/id/scripting/functions/toupper.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+{
+    text[0] = toupper(text[0]);
+    //Ini mengubah karakter utama menjadi huruf besar.
+    return 1;
+}
+```
+
+## Fungsi Terkait
+
+- [tolower](tolower)
```
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/toupper.md:34]: Deskripsinya jangan lupa

File: docs/translations/id/scripting/functions/strcmp.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+title: strcmp
+description: Membanding dua string untuk mengecek apakah mereka sama.
+tags: ["string"]
+---
+
+<LowercaseNote />
+
+## Deskripsi
+
+Membanding dua string untuk mengecek apakah mereka sama.
```
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/strcmp.md:11]: Deskripsinya harusnya sesuai sama yang lain, menggunakan "Fungsi ini", lalu dilanjut dengan "membandingkan kedua string"

**Review by Aiuraa — CHANGES_REQUESTED** [review]
Thanks for your contribution!, there is room for improvement that you can do, and also i saw `# Related Function` is not really been translated to indonesian on some places making it inconsistent.

File: docs/translations/id/scripting/functions/BanEx.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+Function ini tidak me-return value yang spesifik.
+
+## Examples
+
+```c
+public OnPlayerCommandText( playerid, cmdtext[] )
+{
+    if (!strcmp(cmdtext, "/banme", true))
+    {
+        // Bans the player who executed this command and includes a reason ("Request")
```
[the remark this answers is not in the document]
Comment (GNZTMPZ) [reply docs/translations/id/scripting/functions/BanEx.md]: eh mangap, ini kelupaan 😅

File: docs/translations/id/scripting/functions/tolower.md (hunk 6 above)
Comment (GNZTMPZ) [reply docs/translations/id/scripting/functions/tolower.md]: weird, tp sebenarnya ini udah di fix tauu.. kok masi kek gini?

aneh dah gh 🤔

File: docs/translations/id/scripting/functions/tolower.md (hunk 7 above)
Comment (GNZTMPZ) [reply docs/translations/id/scripting/functions/tolower.md]: sebenarnya dr document awalnya gak ada kak, makanya gak dimasukin

namun, klo mau dimasukin yowes tak masukin 👍

File: docs/translations/id/scripting/functions/strcmp.md (hunk 9 above)
Comment (GNZTMPZ) [reply docs/translations/id/scripting/functions/strcmp.md]: sebenarnya dr document awalnya gak ada kak, makanya gak dimasukin

namun, klo mau dimasukin yowes tak masukin 👍

File: docs/translations/id/scripting/functions/toupper.md (hunk 8 above)
Comment (GNZTMPZ) [reply docs/translations/id/scripting/functions/toupper.md]: sebenarnya dr document awalnya gak ada kak, makanya gak dimasukin

namun, klo mau dimasukin yowes tak masukin 👍

**Comment by SanityZeroPercent** [comment]
Hey @leonardssh, sorry for bothering you.. 

We really want to help translating documentations and maintain them. Anyway, I've added my name to the CODEOWNERS, but github said that we don't have write access. I wonder if me, along with @Aiuraa and @rsetiawan7 could have the write access if that's possible. 

Thanks.

**Review by Aiuraa — APPROVED** [review]
I think it's good and can be safe to merge, thanks for your contributions though!

**Review by rsetiawan7 — APPROVED** [review]
lgtm

**Comment by leonardssh** [comment]
Thanks for your contributions!

**Pull request merged**